assets/stylesheets/common/campaign.scss

```diff
@@ -244,6 +245,20 @@ body.archetype-regular {
   }
 }
 
+html:not(.mobile-view) {
```

Comment:
Kinda nitpicking here but why this instead of `html.desktop-view` ?

Reply:
Just following existing pattern that @jordanvidrine had used.

Reply:
That's my fault! `html.desktop-view` is better.